src/Util.java

```diff
@@ -0,0 +1,96 @@
+
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+
+public class Util {
+
+	/**
+	 * Create utility method, which counts even numbers in given collection;
+	 */
+	public int countEvenNumbers(List<? extends Number> list){
+		int count = 0;
+		for (Number number: list) {
+			if(number.intValue() % 2 == 0)
+				count++;
+		}
+		return count;
+	}
+
+	/**
+	 * Create utility method, which takes two lists and index and replaces element at given position
+	 * in the first collection with corresponding element from the second one,
+	 * without possibility to do vice versa;
+	 */
+	public void replaceElement(List list1, List list2, int index){
+		list1.set(index, list2.get(index));
+	}
+
+	/**
+	 * Create utility method, which takes two lists and index and switches elements at given position;
+	 * @param list1 first list
+	 * @param list2 second list
+	 * @param index index of element to switch
+	 */
+	public void switchElements(List list1, List list2, int index){
+		Object o1 = list1.get(index);
+		list1.set(index, list2.get(index));
+		list2.set(index,o1);
+	}
+
+	/**
+	 * Create utility method, which finds maximum element in the given list.
+	 * @param list
+	 * @return maximum element of list
+	 */
+	public Number findMax(List<? extends Number> list){
```

Comment:
It isn't as general, as possible. What if I want to create my own class, specifying how to compare it's instances. I'd expect this method to support my class too.

Reply:
Ok) I'll think how to do it))

Reply:
Take a look at little sister who screams when her drink is worse.